ui/src/pages/backoffice/Series/SeriesSearch/SeriesListEntry.js

```diff
@@ -33,6 +33,17 @@ export class SeriesListEntry extends Component {
 
   renderRelations = () => {
     const { series } = this.props;
+    let literatureNum;
```

Comment:
You can initialize it here `literatureNum = 0` and skip the `else` part in your condition.